Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -139,50 +158,59 @@ public final class ExploreView: UIView {
         setup()
     }
 
+    public init(recommendationsDataSource: ExploreViewRecommendationsDatasource) {
+        self.recommendationsDataSource = recommendationsDataSource
+        super.init(frame: .zero)
+        setup()
+    }
```

Comment:
This method should have the `withAutoLayout` parameter for consistency

Reply:
The init that was being used did not have this parameter, but was using `frame`. Why does this method need the autoLayout parameter if we are not going to use it?

Reply:
Wasn't aware/thinking of the convenience init we were using from FinniverseKit, fixed now.